eval-config.nix

```diff
@@ -82,12 +82,12 @@ let
     modules = modules ++ [ argsModule ] ++ baseModules;
     specialArgs = { modulesPath = builtins.toString ./modules; } // specialArgs;
   });
-in
-
-{
-  inherit (eval._module.args) pkgs;
-  inherit (eval) options config;
-  inherit (eval) _module;
-
-  system = eval.config.system.build.toplevel;
-}
+  
+  withExtraAttrs = module:
```

Comment:
Can you rename `module` to `configuration` and merge this set with `configuration` to match NixOS's implementation:

https://github.com/NixOS/nixpkgs/blob/b1f2198021490b51fb92b8b09db97b9ba2a7b4ce/nixos/lib/eval-config.nix

Reply:
Thanks for your feedback @Enzime !
I fixed it to match NixOS's implementation. But another attributes(`_type`, `class`, `type`) is exposed, is this ok? or should I remove them?
NixOS configuration (`nixpkgs.lib.nixosSystem`) already expose these attributes.